template/greeter/Dockerfile

```diff
@@ -2,10 +2,13 @@
 
 FROM --platform=${BUILDPLATFORM:-<%- platformName %>} node:18-alpine AS dev-deps
 
+RUN corepack enable
+RUN corepack prepare pnpm@latest --activate
```

Comment:
`RUN corepack enable pnpm` seems to do the job of the above two

Reply:
Is there some gotcha around the pnpm version? https://pnpm.io/installation#using-corepack

Reply:
Oooo unsure, I'll experiment

Reply:
<img width="485" alt="image" src="https://github.com/seek-oss/skuba/assets/18017094/6cc81545-79c1-4a2f-b889-069164964029">

Appears to install latest version

Reply:
<img width="495" alt="image" src="https://github.com/seek-oss/skuba/assets/18017094/ae8d54ac-5aa4-44a3-8c64-1308d46d4f2e">

Appears to respect the `packageManager` from package.json